google-cloud-storage/src/test/java/com/google/cloud/storage/ResumableMediaTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.google.cloud.BaseService;
+import com.google.cloud.ExceptionHandler;
+import java.net.URL;
+import org.junit.Assert;
+import org.junit.Test;
+
+public final class ResumableMediaTest {
```

Comment:
This test seems orthogonal to the other changes?

Reply:
This is a related move after separating the open of a resumable session from the BlobWriteChannel. Because there are differing retry handling needs for open vs write on a resumable session and because BlobWriteChannel implements Restorable, I separated open out into ResumableMedia helper class and this tests moved here from BlobWriteChannelTest.

Reply:
Ahh okay got it, I didn't see where this was moved from.